test/module/irohad/ametsuchi/ametsuchi_test.cpp

```diff
@@ -226,12 +223,11 @@ TEST_F(AmetsuchiTest, SampleTest) {
 
   // Block store tests
   auto hashes = {block1.hash(), block2.hash()};
-  validateCalls(blocks->getBlocks(1, 2),
-                [i = 0, &hashes](auto eachBlock) mutable {
-                  EXPECT_EQ(*(hashes.begin() + i), eachBlock->hash());
-                  ++i;
-                },
-                2);
+
+  auto stored_blocks = blocks->getBlocks(1, 2);
+  for (size_t i = 0; i < stored_blocks.size(); i++) {
```

Comment:
Expected size of `stored_blocks` is not checked. Reuse `validateCalls` or remove it.